openlibrary/templates/lib/nav_head.html

```diff
@@ -15,13 +22,23 @@
   $     { "href": "/developers", "text": _("Developer Center") },
   $     { "href": "/help", "text": _("Help & Support") }
   $ ]
+  $ browseLinks = [
+  $     { "href": "/subjects", "text": _("Subjects") },
+  $     { "new": "", "href": "/explore", "text": _("Library Explorer") },
+  $     { "href": "/lists", "text": _("Lists") },
+  $     { "new": "", "href": "/collections", "text": _("Collections") },
+  $     { "href": "/k-12", "text": _("K-12 Student Library") },
+  $     { "href": "/random", "text": _("Random Book") },
+  $     { "href": "/advancedsearch", "text": _("Advanced Search") }
+  $ ]
   $ hamburgerProps = {
   $  'name': 'hamburger',
-  $  'links': loginLinks + moreLinks,
+  $  'links': [ { "subsection": _("My Open Library") } ] + loginLinks +
+  $     [ { "subsection": _("Browse") } ] + browseLinks + [ { "subsection": _("More") } ] + moreLinks,
   $  'image': '/static/images/menu.png',
-  $  'label': 'additional options menu'
+  $  'label': 'additional options menu',
+  $  'mask' : ''
```

Comment:
Let's make this a boolean
```suggestion
  $  'show_mask': True
```